native/src/components/HeaderBox.tsx

```diff
@@ -35,10 +36,16 @@ type HeaderBoxProps = {
 const HeaderBox = ({ goBack, canGoBack = true, text }: HeaderBoxProps): ReactElement => {
   const deviceWidth = useWindowDimensions().width
   const theme = useTheme()
+  const { t } = useTranslation('common')
```

Comment:
:x: I think there is no key `back` in `common` atm since you reverted those changes, right?

Reply:
You're right. Grrrr. That was so annoying to move `back` into `common`. I guess I'll be doing that again.